src/main/resources/application.yml

```
@@ -11,4 +11,8 @@ spring:
         options:
           model: gpt-4.1-mini
   mvc:
-    log-resolved-exception: false
\ No newline at end of file
+    log-resolved-exception: false
+jwt:
+  secret: ${JWT_SECRET_KEY}
+  access-token-validity-in-seconds: 1800
+  refresh-token-validity-in-seconds: 86400
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)